sim_pipeline/image_simulation.py

```diff
@@ -100,32 +99,6 @@ def sharp_image(
     return image
 
 
-def galsimobj(image, pix_scale, flux):
```

Comment:
you removed this definition but did not change any other code. Was this definition unused or perhaps used in a jupyter notebook?

Reply:
This was used in jupyter notebook. I removed everything related to this.